tests/bunit.web.query.tests/Labels/LabelQueryExtensionsTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Bunit.TestAssets.BlazorE2E;
+
+namespace Bunit.Labels;
+
+public class LabelQueryExtensionsTests : TestContext
+{
+	public static IEnumerable<object[]> HtmlElementsThatCanHaveALabel =>
+		new List<object[]>
+		{
+			new object[] { "input" },
+			new object[] { "select" },
+			new object[] { "button" },
+			new object[] { "meter" },
+			new object[] { "output" },
+			new object[] { "progress" },
+		};
```

Comment:
I prefer to use the TheoryData type from xUnit, to get a more concise and strongly typed experience. Also, turn into a initialized property so the type isn't created everytime the property is accessed.

```suggestion
	public static TheoryData<string> HtmlElementsThatCanHaveALabel { get; } = new()
	{
		"input",
		"select",
		"button",
		"meter",
		"output",
		"progress",
	};
```

Reply:
TIL about `TheoryData`!